FestivalAttendeeThread.java

```diff
@@ -0,0 +1,38 @@
+package concurrency;

```

Comment:
you are still having problems with the packages and missing imports, please try to fix them (it is important for the final project)